frontend/src/views/web/challenge/submission.html

```diff
@@ -227,7 +227,7 @@ <h5 class="w-300">Make Submission</h5>
                                         <span>Upload file</span>
                                         <input type="file" ngf-select ng-model="challenge.input_file"
                                             name="challenge.input_file"
-                                            accept=".json, .zip, .txt, .tsv, .gz, .csv, .h5, .npy">
+                                            accept=".json, .zip, .txt, .tsv, .gz, .csv, .h5, .npy, .npz">
```

Comment:
There is a field in the backend using which we can control it. Can you please change it to that?

Reply:
@RishabhJain2018 We want to add `.npz` manually for all phases of open catalyst challenge right?

Reply:
Yes, but in `accept` we want that fields should come from backend rather than being hardcoded as it is now.